ironic/resource_ironic_node_v1.go

```diff
@@ -145,6 +148,25 @@ func resourceNodeV1() *schema.Resource {
 				// This did not change if the current provision state matches the target
 				DiffSuppressFunc: targetStateMatchesReality,
 			},
+			"power_state": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"target_power_state": {
+				Type:     schema.TypeString,
+				Optional: true,
+
+				// If power_state is same as target_power_state, we have no changes to apply
+				DiffSuppressFunc: func(_, old, new string, d *schema.ResourceData) bool {
+					return new == d.Get("power_state").(string)
+				},
+			},
+			"power_state_timeout": {
+				Type:         schema.TypeString,
```

Comment:
Why a string? Is it some terraform limitation?

Reply:
Terraform's type system is extremely limited... the next version is better, but 0.11.3 does have TypeInt, I've fixed it.